src/view/visualizer/sketch/index.js

```diff
@@ -1,195 +1,195 @@
         }
 
         if (changeWidth) {
-          p.background('white')
-          reDrawKnown()
+          p.background('white');
+          reDrawKnown();
         }
 
         objs.forEach(function(variable) {
-            if (update && variable.name === val) {
-              variable.highlight = true
-              variable.color = fillColor
-              console.log(state[val])
-              variable.value = state[val]
+          if (update && variable.name === val) {
+            variable.highlight = true;
+            variable.color = fillColor;
+            console.log(state[val]);
+            variable.value = state[val];
 
-            } else {
-                variable.highlight = false
+          } else {
+            variable.highlight = false;
 
-            }
+          }
 
-            variable.width = rectWidth
-            variable.height = rectHeight
-            variable.display()
-        })
+          variable.width = rectWidth;
+          variable.height = rectHeight;
+          variable.display();
+        });
 
-        known[val] = state[val]
+        known[val] = state[val];
       }
-    })
-  }
+    });
+  };
 
   p.windowResized = () => {
-    resizeCanvasToVisualizer()
-    overflow = false
-    p.background('white')
-    reDrawKnown()
-  }
-  
+    resizeCanvasToVisualizer();
+    overflow = false;
+    p.background('white');
+    reDrawKnown();
+  };
+
   function reDrawKnown() {
-    let xCoord = 50
-    let yCoord = 0
-    let curObj = -1
+    let xCoord = 50;
+    let yCoord = 0;
+    let curObj = -1;
     objs.forEach((variable) => {
-      variable.width = rectWidth
-      variable.height = rectHeight
-      curObj += 1
-      yCoord = (rectHeight * 2) * curObj + rectHeight
+      variable.width = rectWidth;
+      variable.height = rectHeight;
+      curObj += 1;
+      yCoord = (rectHeight * 2) * curObj + rectHeight;
 
       if (yCoord > p.height - (rectHeight + 15)) {
-        yCoord = rectHeight
-        xCoord += rectWidth + 20
-        curObj = 0
+        yCoord = rectHeight;
+        xCoord += rectWidth + 20;
+        curObj = 0;
       }
 
       if (xCoord > p.width - (rectWidth + 20)) {
-          overflow = true
-          return
+        overflow = true;
+        return;
       }
 
-      variable.x = xCoord
-      variable.y = yCoord
-      variable.display()
-    })
+      variable.x = xCoord;
+      variable.y = yCoord;
+      variable.display();
+    });
   }
 
   function getColor(colorState) {
     switch (colorState) {
-      case 0: return '#00c9fe'
-      case 1: return '#00d4df'
-      case 2: return '#00dab4'
-      case 3: return '#ffb5b4'
-      case 4: return '#ff96b4'
-      case 5: return '#0096b4'
-      case 6: return '#0069fb'
-      case 7: return '#af96fb'
-      case 8: return '#ff8b6a'
-      case 9: return '#ffbf00'
-      case 10: return '#ff00c3'
-      case 11: return '#0000c3'
-      default: return '#00c9fe'
+    case 0: return '#00c9fe';
```

Comment:
Do we want to remove the indent from the switch case?

Reply:
I have no opinion on this - @zsmoore @noahpoz @AlexPerri